mailgun-relayery-init/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/Parquery/mailgun-relayery/database"
+	ver "github.com/Parquery/mailgun-relayery/version"
+)
+
+var version = flag.Bool("version", false,
+	"print the version to STDOUT and exit immediately")
+
+var databaseDir = flag.String("database_dir", "",
+	"Path to the directory where the database should be initialized")
+
+func main() {
+	os.Exit(func() (retcode int) {
+		flag.Parse()
+
+		if *version {
+			fmt.Println(ver.Latest)
+			return 0
+		}
+
+		logOut := log.New(os.Stdout, "[database init] ", log.Ldate|log.Ltime)
```

Comment:
maybe just `[init]` since "database" is not mentioned in the name of the utility?